ts/features/fastLogin/saga/tokenRefreshSaga.ts

```diff
@@ -0,0 +1,133 @@
+import { SagaIterator } from "redux-saga";
+import { call, delay, put, takeLatest } from "typed-redux-saga/macro";
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import { ValidationError } from "io-ts";
+import {
+  refreshSessionToken,
+  sessionExpired
+} from "../../../store/actions/authentication";
+import { SessionToken } from "../../../types/SessionToken";
+import { startApplicationInitialization } from "../../../store/actions/application";
+import {
+  createFastLoginClient,
+  createNonceClient,
+  performFastLogin,
+  performGetNonce
+} from "../backend";
+import { apiUrlPrefix } from "../../../config";
+import { SagaCallReturnType } from "../../../types/utils";
+import { LollipopConfig } from "../../lollipop";
+import { getKeyInfo } from "../../lollipop/saga";
+import {
+  FastLoginResponse,
+  FastLoginResponseType,
+  NonceBaseResponseType,
+  NonceResponse
+} from "../backend/mockedFunctionsAndTypes";
+
+export function* watchTokenRefreshSaga(): SagaIterator {
+  yield* takeLatest(refreshSessionToken.request, doRefreshTokenSaga);
+}
+
+type RequestStateType = {
+  counter: number;
+  success: boolean;
+  error: string | undefined;
+};
+
+const MAX_RETRYES = 3;
```

Comment:
Even if we already don't know how to manage this. Either by a remote config (io-services-metadata) or not, I suggest to put this in our `.env.*` environment files and retrieve this value from configs. Later, if we adopt a remote approach we could use this value as a fallback anyway. What do you think about that?

Reply:
Or if you prefer to leave it the way it is, we could just add a comment and track it on with a JIRA ticket.